docs/integrations/google_colab.rst

```diff
@@ -3,38 +3,129 @@
 Neptune-Google Colab Integration
 ================================
 
-You can run experiments on Google Colab and track them with Neptune.
+|Google Colab| is a temporary runtime environment. This means you lose all your data (unless saved externally) once you restart your kernel. 
 
-Follow these steps:
+This is where you can leverage Neptune. By running your experiments on Google Colab and tracking them with Neptune, you can log 
 
-1. Install Neptune client:
+* parameters
+* metrics and losses
+* hardware consumption, and
+* model checkpoints and other artifacts
```

Comment:
I'd add a bullet talking about downloading model checkpoints directly into Colab. 

I think this is a real value for the user and it should be highlighted.

Reply:
Added a line linking to the download guide doc after the bullets